### DISCUSSION TITLE
modify jspsych.css to add background image

### DISCUSSION THREAD

**Discussion by valentinavellani** [jspsych/jsPsych#1133] [Q&A]
Hi All,
I am coding my task using Gorilla and I would like to add a backgroung image to the task.
I thought one way to do so would be to modify the jspsych.css file that provides easthetic for the whole task.
Do you have any idea on how to do so? 

Best
Valentina

**Comment by becky-gilbert** [comment]
Hi @valentinavellani, you can add some custom CSS to the page that sets the 'background-image' property for the `<body>` element. One way to do add the CSS is to put it inside `<style></style>` tags inside the `<head>` section of the page, like this:
```html
<head>
    <script src="jspsych-6.1.0/jspsych.js"></script>
    <script src="jspsych-6.1.0/plugins/jspsych-survey-text.js"></script>
    <script src="jspsych-6.1.0/plugins/jspsych-html-button-response.js"></script>
    <link rel="stylesheet" href="jspsych-6.1.0/css/jspsych.css">
    <style>
        body {
            background-image: url("jspsych-6.1.0/examples/img/1.gif");
        }
    </style>
</head>
```
You might also want look at other CSS properties like 'background-repeat', 'background-size', and 'background-position'. You can see some options and examples on [this page about background-image](https://www.w3schools.com/cssref/pr_background-image.asp).
